src/content/developers/docs/scaling/index.md

```diff
@@ -92,6 +92,11 @@ Learn more about [Validium](/developers/docs/scaling/validium/).
 - Not all solutions require utilizing the Ethereum consensus algorithm directly, and alternatives can offer benefits that would otherwise be difficult to obtain.
 - No one scaling solution is enough to fulfill the [Ethereum vision](/upgrades/vision/).
 
+## Why not just stack scaling solutions? {#why-not-stack-scaling}
```

Comment:
I'd like to include the article for reading, but I think this section is a bit premature given we are still working on getting layer 2's online in their full form. As the post says "it seems premature to settle definitions while the architecture of the multi-rollup ecosystem is far from set in stone and most of the discussion is happening only in theory."

Reply:
Fair enough! I think the content itself answers an important point about Layer 2s if the naming can be pushed back for future discussions.